pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -1070,44 +1083,45 @@ private void onReturnFailure(String errorMsg, Exception e) {
    */
   public Pair<Map<InstancePartitionsType, InstancePartitions>, Boolean> getInstancePartitionsMap(
       TableConfig tableConfig, boolean reassignInstances, boolean bootstrap, boolean dryRun) {
-    return getInstancePartitionsMap(tableConfig, reassignInstances, bootstrap, dryRun, false);
+    return getInstancePartitionsMap(tableConfig, reassignInstances, bootstrap, dryRun, false,
+        new TableRebalanceLogger(LOGGER));
```

Comment:
nit: add a comment mentioning that this is called from a code path outside of rebalance?